src/platform.config.ts

```diff
@@ -4,11 +4,27 @@ import { PiecesClient } from "pieces-copilot-sdk";
 let BASE_URL: string;
 let WS_URL: string;
 
-// Define your port here 
+// Define your port based on the operating system
+let PORT: number;
+
+// Define your port here
 // LINUX PORT=5323
 // For Windows and MacOs PORT=1000
 
-const PORT = 5323
+const platform = navigator.platform.toLowerCase();
+console.log("OS TYPE IS: ",platform)
+
+const userAgent = navigator.userAgent.toLowerCase();
```

Comment:
Is this required? 

I don't find any use case of this as of now.

Reply:
Removed it.